scripts/resources/Suwayomi Server.bat

```diff
@@ -0,0 +1,22 @@
+@echo off
+set auto_update=1
+
+:: Move CWD to script folder
+pushd "%~0\.."
+
+:: Auto-Updater
+if %auto_update% NEQ 1 goto :launch
+for /f "tokens=2" %%i in ('curl -s https://api.github.com/repos/Suwayomi/Suwayomi-Server-preview/releases/latest^|findstr "https://.*jar"') do (
+	pushd bin
+	if NOT EXIST "%%~nxi" (
+    echo Updating to %%~ni
+		del /q "*.jar"
+		curl -sL %%i -o "%%~nxi"
+		mklink /h "Suwayomi-Server.jar" "%%~nxi"
+	)
+	popd
+)
+
+:launch
+start "" jre\bin\javaw %* -jar bin\Suwayomi-Server.jar
```

Comment:
you can (in preview for now) launch the server using the launcher and the `--launch` argument
so you don't need to run it directly like this anymore

the advantage is that the launcher deals with root directories other than the default

Reply:
It is redirecting additional parameters with `%*` so it'll support that being configured with `-Dsuwayomi.tachidesk.config.server.rootDir=...` as [the wiki suggests](https://github.com/Suwayomi/Suwayomi-Server/wiki/The-Data-Directory#custom)

Reply:
What Robo means is that the launcher has its own config saved thats needed for people to always get the right settings, and the new --launch parameter will directly start the server without the launcher opening
```suggestion
start "" jre\bin\javaw %* -jar Suwayomi-Launcher.jar --launch
```

Reply:
Fair enough. Ideally the auto-updater would be built into the launcher, but I lack the Java-fu for that so batch it is 😅